Add python-glmnet recipe

jseabold opened the pull request.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/python-glmnet```) and found some lint.

Here's what I've got...


For **recipes/python-glmnet**:

 * The recipe could do with some maintainers listed in the `extra/recipe-maintainers` section.
 * The recipe must have a `build/number` section.
 * The recipe `license` should not include the word "License".

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/python-glmnet```) and found some lint.

Here's what I've got...


For **recipes/python-glmnet**:

 * The top level meta keys are in an unexpected order. Expecting ['package', 'source', 'build', 'requirements', 'test', 'about', 'extra'].

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/python-glmnet```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/python-glmnet```) and found some lint.

Here's what I've got...


For **recipes/python-glmnet**:

 * Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/python-glmnet```) and found it was in an excellent condition.

synapticarbors commented on recipes/python-glmnet/meta.yaml:
```diff
@@ -0,0 +1,40 @@
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/civisanalytics/python-glmnet.git
```
I believe the preference is to get the source from pypi or the github release archive rather than a git tag:

http://conda-forge.github.io/docs/meta.html#build-from-tarballs-not-repos

synapticarbors commented on recipes/python-glmnet/meta.yaml:
```diff
@@ -0,0 +1,40 @@
+
+about:
+  home: https://github.com/civisanalytics/python-glmnet
+  license: GPL 2
```
`GPL-2.0`

synapticarbors commented on recipes/python-glmnet/meta.yaml:
```diff
@@ -0,0 +1,40 @@
+about:
+  home: https://github.com/civisanalytics/python-glmnet
+  license: GPL 2
+  summary: 'Python wrapper for glmnet'
```
It looks like there is a license file, so please add:

```
license_file: LICENSE
```

Also please add `dev_url: https://github.com/civisanalytics/python-glmnet`

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/python-glmnet```) and found some lint.

Here's what I've got...


For **recipes/python-glmnet**:

 * Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/python-glmnet```) and found it was in an excellent condition.

jseabold commented:
@synapticarbors Addressed the code comments and removed the windows builds. Can't seem to find a successful recipe that builds a package with a fortran dependency on windows, and I'm not terribly interested in futzing with this on a first pass.

jakirkham commented on recipes/python-glmnet/meta.yaml:
```diff
@@ -0,0 +1,48 @@
+
+extra:
+  recipe-maintainers:
+    jseabold
```
FYI this was missing a `-` before it. Have raised an issue in the feedstock regarding getting this fixed.

xref: https://github.com/conda-forge/glmnet-feedstock/issues/1
